src/lib/util/Config.ts

```diff
@@ -119,6 +130,17 @@ export class Config {
 		return !e || e.disabled;
 	}
 
+	public isDepCheckerEnabled(name: string): boolean {
+		if (!this.configContent.depCheckers) {
+			// Fast exit. No definitions means all are enabled.
+			return true;
+		}
+
+		const d = this.configContent.depCheckers.find(d => d.name === name);
```

Comment:
In preparation for other types of config attributes, it is probably worth breaking this out into a `getDepCheckerConfig` method.